src/tcas/src/lib/TcasConstants.ts

```diff
@@ -0,0 +1,680 @@
+/* eslint-disable camelcase */
+const REFRESH_RATE: number = 1000; // Time between refreshes in ms
+const TRACKING_MAX: number = 40; // # max contacts tracked - AMM 34-43-00:6a
+const DISPLAY_MAX: number = 8; // # max contacts tracked - usually configurable by PIN program
+const MEMORY_MAX: number = 200; // Max history before culling
+const MIN_VS: number = -6000;
+const MAX_VS: number = 6000;
+const INHIBIT_CLB_RA: number = 39000; // for all climb RA's
+const INHIBIT_INC_DES_RA_AGL: number = 1450; // for increase descent RA's
+const INHIBIT_ALL_DES_RA_AGL: number = 1200; // 1200 takeoff, 1000 approach
+const INHIBIT_ALL_RA: number = 1000; // 1100 in climb, 900 in descent
+const REALLY_BIG_NUMBER: number = 1000000;
+const INITIAL_DELAY: number = 5; // in seconds
+const FOLLOWUP_DELAY: number = 2.5; // in deconds
+const INITIAL_ACCEL: number = 8.04; // 0.25G in f/s^2
+const FOLLOWUP_ACCEL: number = 10.62; // 0.33G in f/s^2
+const TA_EXPIRATION_DELAY: number = 4; // in seconds
+const MIN_RA_DURATION: number = 5; // in seconds
+const VOL_BOOST: number = 1.25; // multiplier
+const CLOSURE_RATE_THRESH: number = -40; // in knots
+
+export enum TcasThreat { THREAT = 0, ALL = 1, ABOVE = 2, BELOW = 3 }
+export enum TcasState { NONE = 0, TA = 1, RA = 2 }
+export enum TcasMode { STBY = 0, TA = 1, TARA = 2 }
+export enum XpdrMode { STBY = 1, ON = 3, ALT = 4, }
+export enum TaRaIndex { TA = 0, RA = 1 }
+export enum TaRaIntrusion { TRAFFIC = 0, PROXIMITY = 1, TA = 2, RA = 3 }
+export enum Intrude { RANGE = 0, ALT = 1, SPEED = 2}
+export enum RaSense { UP = 0, DOWN = 1 }
+export enum RaType { CORRECT = 0, PREVENT = 1 }
+export enum Limits { MIN = 0, MAX = 1 }
+export enum Inhibit { NONE = 0, ALL_RA_AURAL_TA = 1, ALL_RA = 2, ALL_CLIMB_RA = 3, ALL_DESC_RA = 4, ALL_INCR_DESC_RA = 5 }
+
+export interface JS_NPCPlane {
+    name: string,
+    uId: number,
+    lat: number,
+    lon: number,
+    alt: number,
+    heading: number
+}
+
+interface AltitudeMatrix {
+    readonly 3: readonly [number, number],
+    readonly 4: readonly [number, number],
+    readonly 5: readonly [number, number],
+    readonly 6: readonly [number, number],
+    readonly 7: readonly [number, number],
+}
+
+interface SensitivityMatrix {
+    readonly 1: readonly [number, number],
+    readonly 2: readonly [number, number],
+    readonly 3: readonly [number, number],
+    readonly 4: readonly [number, number],
+    readonly 5: readonly [number, number],
+    readonly 6: readonly [number, number],
+    readonly 7: readonly [number, number],
+    readonly 8: readonly [number, number],
+}
+
+interface SensitivityList {
+    readonly 1: number,
+    readonly 2: number,
+    readonly 3: number,
+    readonly 4: number,
+    readonly 5: number,
+    readonly 6: number,
+    readonly 7: number,
+    readonly 8: number,
+}
```

Comment:
Reading up on this a bit[^1] specifically sensitivity level 1 is important for the FWS to flag the TCAS being in standby for any reason, as the FWC will use this to trigger the `TCAS STBY` memo on the EWD (note: this is different from the `NAV TCAS STBY` warning which you already adjusted and also different from `TCAS FAULT`). 

This means you to set up the ECAM memo accurately it should be updated to check for your new LVar `A32NX_TCAS_SENSITIVITY == 1` (ARINC word 274, bits 25-23[^2] with SL = Sensitivity Level)

Finally I wonder what exactly happens when ATC Altitude Reporting has been set to off (#5089) - does this simply cause the sensitivity level to go to 1, triggering the amber TCAS STBY memo?

[^1]: From https://skybrary.aero/sites/default/files/bookshelf/106.pdf (page 84):
![image](https://user-images.githubusercontent.com/929769/143660018-a09d72c5-6b29-4388-a6ea-28f3b83f5a6f.png)

[^2]: From https://manualzz.com/doc/22316884/arinc-specification-429p2-16 (page 131):
![image](https://user-images.githubusercontent.com/929769/143660196-d808ea2b-71b1-42a6-bc41-cb49ac4f6072.png)

Reply:
As this is not there yet I would vote for it to be added in a later PR.

Reply:
Yes this would be a fairly simple modification for a future PR

Reply:
> As this is not there yet I would vote for it to be added in a later PR.

I'm not sure what you mean - the sensitivity seems to have been added? The only missing thing is changing the current ECAM to actually use that rather than looking at the TCAS switch (changing the SimVar in the ECAM is a one liner).
In fact doing so might be very illuminating because it's another signal that can be used to check that the sensitivity is implemented correctly. If updating the data source in the ECAM makes it appear at the wrong times, that then may indicate that the actual logic is not quite correct.

So I think there's a lot of benefit in hooking it up now, rather than merging this and then in future someone has to come in and then if it's not quite accurate try to understand why the TCAS is behaving the way it is.

Reply:
Ok, now I get it. I thought it‘s about a new warning (I searched for it already and didn‘t find it). So your suggestion is to change the old for „NAV TCAS FAULT“ to use sensitivity instead of switch position?

Reply:
Maybe just suggest the change…that would be nice ;-)

Reply:
Ah, finally I can respond here again, because I had a review comment pending. Yeah, any objection to me pushing it directly to this branch? It's a minor change to UpperECAM, but I can't suggest it on GitHub because this PR did not touch that part of the file.